src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1273,6 +1292,11 @@ void lv_vg_lite_flush(struct _lv_draw_vg_lite_unit_t * u)
         LV_PROFILER_DRAW_END;
         return;
     }
+
+    /* When the GPU is in idle state, it means that the GPU is not accessing
+     * any resources and the cache reference state can be released.
+     */
+    lv_vg_lite_clear_all_pending(u);
```

Comment:
Question: The fact of the GPU being idle would not mean it will not access the same cache entries again, so why not keep then and just invalidate upon an explicit "cache miss"?

Reply:
The cache entry needs to be locked while the GPU is reading the image resource, because the GPU only records the pointer of the image instead of copying the image, so it is necessary to prevent the cache manager from mistakenly kicking out the image resource that the GPU is referencing.
The meaning of pending is to record the locked cache entry in the list, and when the GPU completes the work, the cache is unlocked uniformly, and the cache is not cleared.

Reply:
Makes sense now, thank you for clarification!